Packages/com.llealloo.audiolink/Runtime/Scripts/AudioReactivePostProcessing.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+
+namespace AudioLink
+{
+#if UDONSHARP
+    using UdonSharp;
+    using UnityEngine.Rendering.PostProcessing;
+
+    public class AudioReactivePostProcessing : UdonSharpBehaviour
+#else
+    public class AudioReactivePostProcessing : MonoBehaviour
```

Comment:
Use [RequireComponent(typeof(PostProcessVolume))] ?

Reply:
PostProcessingVolume is now required